internal/runners/export/log.go

```diff
@@ -85,3 +100,31 @@ func (l *Log) Run(params *LogParams) (rerr error) {
 
 	return nil
 }
+
+func ignoreLogFile(logFile string) (bool, error) {
+	file, err := os.Open(logFile)
+	if err != nil {
+		return false, errs.Wrap(err, "failed to open log file")
+	}
+	defer file.Close()
+
+	regex := regexp.MustCompile(`.*\] Args: \[(.*?)\], Flags: \[.*?\]`)
```

Comment:
```suggestion
	regex := regexp.MustCompile(`Args: \[(.*?)\], Flags: \[.*?\]`)
```

Doesn't seem necessary for this function to have an opinion on what the prefix looks like.